command/agent/http_test.go

```diff
@@ -476,29 +477,36 @@ func TestParseWait_InvalidIndex(t *testing.T) {
 func TestParseConsistency(t *testing.T) {
 	ci.Parallel(t)
 	var b structs.QueryOptions
+	var resp *httptest.ResponseRecorder
 
-	req, err := http.NewRequest("GET",
-		"/v1/catalog/nodes?stale", nil)
-	if err != nil {
-		t.Fatalf("err: %v", err)
+	testCases := [2]string{"/v1/catalog/nodes?stale", "/v1/catalog/nodes?stale=true"}
+	for _, url := range testCases {
+		req, err := http.NewRequest("GET", url, nil)
+		must.NoError(t, err)
+		resp = httptest.NewRecorder()
+		parseConsistency(resp, req, &b)
+		must.True(t, b.AllowStale)
 	}
 
-	parseConsistency(req, &b)
-	if !b.AllowStale {
-		t.Fatalf("Bad: %v", b)
-	}
+	req, err := http.NewRequest("GET", "/v1/catalog/nodes?stale=false", nil)
+	must.NoError(t, err)
+	resp = httptest.NewRecorder()
+	parseConsistency(resp, req, &b)
+	must.True(t, !b.AllowStale)
+
+	req, err = http.NewRequest("GET", "/v1/catalog/nodes?stale=random", nil)
+	must.NoError(t, err)
+	resp = httptest.NewRecorder()
+	parseConsistency(resp, req, &b)
+	must.EqOp(t, resp.Code, 400)
```

Comment:
The expectation (the `400` here) goes first

(it only matters when rendering failure output, but still it's nice to avoid causing confusion)

Reply:
Do you mean we should add in extra assertion like this to provide consistency to the test case? 
```
	must.False(t, b.AllowStale)
	must.EqOp(t, resp.Code, 400)

```

Reply:
Oh sorry, just mean you have the order of the arguments backwards for EqOp(t, expectation, value): https://pkg.go.dev/github.com/shoenig/test/must#EqOp

should be `EqOp(t, 400, resp.Code)`